installing/installing-preparing.adoc

```diff
@@ -62,6 +62,7 @@ For installer-provisioned infrastructure installations, you can use an existing
 
 You can use the installer-provisioned infrastructure method to create appropriate machine instances on your hardware for xref:../installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[vSphere], and xref:../installing/installing_bare_metal_ipi/ipi-install-overview#ipi-install-overview[bare metal]. Additionally, for xref:../installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.adoc#installing-vsphere-installer-provisioned-network-customizations[vSphere], you can also customize additional network parameters during installation.
 
+For some installer-provisioned infrastructure installations, for example on the {vmw-first} and bare metal platforms, the external traffic that reaches the ingress virtual IP is not balanced between the default `IngressController` replicas. For {vmw-short}  and bare metal installer-provisioned infrastructure installations where it is required to exceed the baseline `IngressController` router performance, you must configure an external load balancer. Configuring an external load balancer achieves the performance of multiple `IngressController` replicas. For more information about the baseline `IngressController` performance, see xref:../scalability_and_performance/optimization/routing-optimization.adoc#baseline-router-performance_routing-optimization[Baseline Ingress Controller (router) performance]. For more information about configuring an external load balancer, see xref:../installing//installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#nw-osp-configuring-external-load-balancer_ipi-install-installation-workflow[Configuring a user-managed load balancer].
```

Comment:
1. ingress virtual IP > ingress virtual IP (VIP)
2. where it is required to exceed the baseline `IngressController` router performance > where exceeding the baseline `IngressController` router performance is expected

Reply:
/remove-label peer-review-in-progress

/label peer-review-done